README.md

```diff
@@ -1,6 +1,6 @@
 # GitHub public feedback discussions
 
-In this repository, you can find the [official GitHub public feedback discussions](https://https://github.com/github-community/community/discussions) for the following product areas, as well as an overall category for general product feedback:
+In this repository, you can find the [official GitHub public feedback discussions](https://github.com/github-community/community/discussions) for the following product areas, as well as an overall category for general product feedback:
 
 | **Feedback Category** | **About the Product** 	|
```

Comment:
bitcoin wallet bc1qj6jv4c84fje258jumtv93md5vfnc045kw60zn0

Reply:
In this repository, you can find the [official GitHub public feedback discussions](https://https://github.com/github-community/community/discussions) for the following product areas, as well as an overall category for general product feedback:
In this repository, you can find the [official GitHub public feedback discussions](https://github.com/github-community/community/discussions) for the following product areas, as well as an overall category for general product feedback: